code-name-taran/src/com/spaghettisoft/component/chess/board/ChessBoardManager.java

```diff
@@ -0,0 +1,65 @@
+package com.spaghettisoft.component.chess.board;
+
+import javax.naming.InvalidNameException;
+
+import com.spaghettisoft.component.chess.figures.AbstractFigure;
+import com.spaghettisoft.component.chess.figures.Bishop;
+import com.spaghettisoft.component.chess.figures.IFigures;
+import com.spaghettisoft.component.chess.figures.King;
+import com.spaghettisoft.component.chess.figures.Knight;
+import com.spaghettisoft.component.chess.figures.Pawn;
+import com.spaghettisoft.component.chess.figures.Queen;
+import com.spaghettisoft.component.chess.figures.Rook;
+import com.spaghettisoft.component.chess.figures.AbstractFigure.Color;
+
+public class ChessBoardManager {
+
+	ChessBoard chessBoard;
+
+	public ChessBoardManager(ChessBoard chessBoard) {
+		this.chessBoard = chessBoard;
+	}
+
+	public void fillChessBoardWithFigures() {
+		try {
+			setFirstRowFigures(1, Color.BLACK);
+			setFirstRowFigures(8, Color.WHITE);
+			for (int i = 0; i < this.chessBoard.getBoardLenght(); i++) {

```

Comment:
You Don't need two loops here. The "j" loop is enough, "i" should be kept constants. Remove the "ifs" and put 6 and 2 when calling the methods. Also you can extract the whole loop in separate method. It will make the code more readable.